test/lib.bash

```diff
@@ -389,13 +389,14 @@ function create_htpasswd_users {
   if kubectl get secret htpass-secret -n openshift-config -o jsonpath='{.data.htpasswd}' 2>/dev/null | base64 -d > users.htpasswd; then
     logger.info 'Secret htpass-secret already existed, updating it.'
     # Add a newline to the end of the file if not already present (htpasswd will butcher it otherwise).
+    # shellcheck disable=SC1003
     sed -i -e '$a\' users.htpasswd
```

Comment:
I saw that. I don't know it that a problem or not. It looks weird, like a bug.

Warren wrote that. @warrenvw, maybe you remember what that suppose to achieve?

Reply:
Well it achieves what the comment states. A quick google told me that this is indeed the notation for that in `sed` (I doublechecked this too)

Reply:
Maybe better would be to rewrite that to simpler and POSIX compliant alternative: https://unix.stackexchange.com/a/263965/145501

Reply:
Indeed, changing!